src/main/java/org/folio/models/PoLineInvoiceLineHolder.java

```diff
@@ -5,23 +5,24 @@
 
 import org.folio.rest.acq.model.invoice.InvoiceLine;
 import org.folio.rest.jaxrs.model.CompositePoLine;
+import org.folio.rest.jaxrs.model.PoLine;
 
 import io.vertx.core.json.JsonObject;
 
 
 public class PoLineInvoiceLineHolder {
-  private CompositePoLine poLine;
-  private JsonObject poLineFromStorage;
+  private CompositePoLine poLineFromRequest;
+  private PoLine poLineFromStorage;
   private List<InvoiceLine> invoiceLines;
   private List<InvoiceLine> openOrReviewedInvoiceLines;
-  private List<InvoiceLine> currentYearPaidInvoiceLines;
+  private List<InvoiceLine> currentYearPaidOrCancelledInvoiceLines;
```

Comment:
I think to simplify naming we can use paidOrCancelledInvoiceLines name

Reply:
Done